src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -304,11 +305,11 @@ private void filmSuchen1(final String strUrlFeed, final String thema, final bool
     }
 
     private void filmSuchen2(final String urlFilm_, final String thema, final String titel,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 38 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)